src/commonMain/kotlin/exchange.dydx.abacus/output/input/TransferInput.kt

```diff
@@ -571,6 +571,7 @@ data class TransferInput(
     val summary: TransferInputSummary?,
     val resources: TransferInputResources?,
     val requestPayload: TransferInputRequestPayload?,
+    val cctp: Boolean,
```

Comment:
This won't be needed.  There is already an extension property that does this:

`val TransferInput.isCctp: Boolean`

It looks at the TransferInput's token and chain to determine if the route is CCTP.